include/coap2/coap.h.windows

```diff
@@ -1,60 +0,0 @@
-/*
- * coap.h -- main header file for CoAP stack of libcoap
- *
- * Copyright (C) 2010-2012,2015-2017 Olaf Bergmann <bergmann@tzi.org>
- *               2015 Carsten Schoenert <c.schoenert@t-online.de>
- *
```

Comment:
Deleting this file will break the previous Visual Studio build using the current project files.  I agree that perhaps a better place for this file is in the win32/ directory

Reply:
Or dropping those as visual studio now supports cmake out of the box